src/textual/widget.py

```diff
@@ -515,6 +515,8 @@ def _arrange(self, size: Size) -> DockArrangeResult:
     def _clear_arrangement_cache(self) -> None:
         """Clear arrangement cache, forcing a new arrange operation."""
         self._arrangement_cache.clear()
+        self._stabilized_scrollbar_size = None
+        self._scrollbar_stabilizer = 0
```

Comment:
Consider adding docstrings to these two attributes.
`self._stabilized_scrollbar_size` is already typed in the body of `Widget` but `_scrollbar_stabilizer` is brand new.